bin/cliain/src/snark_relations/relations.rs

```diff
@@ -201,7 +265,7 @@ impl ConstraintSynthesizer<CircuitField> for RelationArgs {
                 .generate_constraints(cs),
 
             RelationArgs::LinearEquation { a, x, b, y } => {
-                LinearEquationRelationWithFullInput::new(a, x, b, y).generate_constraints(cs)
+                LinearEquationRelationWithFullInput::new(a, b, y, x).generate_constraints(cs)
```

Comment:
was the order wrong before and you fixed it or did sth else happen here?

Reply:
the order was wrong (the proper one changed when the relation was made with macro)